addons/website_slides/controllers/main.py

```diff
@@ -476,18 +476,19 @@ def slide_channel_tag_search_read(self, fields, domain):
 
     @http.route('''/slides/slide/<model("slide.slide", "[('website_id', 'in', (False, current_website_id))]"):slide>''', type='http', auth="public", website=True)
     def slide_view(self, slide, **kwargs):
+        is_fullscreen = 'fullscreen' in kwargs
```

Comment:
Should probably check kwargs.get('fullscreen', 0) (depends on 'fullscreen' parameter type).
Because right now if you try to call the route with a falsy value for fullscreen it will still be considered as fullscreen.